mesheryctl/internal/cli/root/completion.go

```diff
@@ -77,7 +77,7 @@ var completionCmd = &cobra.Command{
 		case "fish":
 			return cmd.Root().GenFishCompletion(os.Stdout, true)
 		default:
-			return errors.New("shell not supported")
+			return errors.New("shell not supported, requires one of [bash|zsh|fish]")
```

Comment:
Could we make it more meaningful? Like....

```suggestion
			return errors.New("shell not supported, requires one of the shells eg:[bash|zsh|fish]")
```

// @harkiratsm @leecalcote